src/main/java/com/google/api/codegen/transformer/java/JavaGapicSamplesPackageTransformer.java

```diff
@@ -66,11 +67,18 @@ public JavaGapicSamplesPackageTransformer(PackageMetadataConfig packageConfig) {
   }
 
   private List<PackageDependencyView> additionalSampleDependencies() {
-    return Collections.singletonList(
+    PackageDependencyView apiCommonDep =
         PackageDependencyView.newBuilder()
             .group("com.google.api")
             .name("api-common")
             .versionBound(packageConfig.apiCommonVersionBound(TargetLanguage.JAVA))
-            .build());
+            .build();
+    PackageDependencyView commonsCliDep =
+        PackageDependencyView.newBuilder()
+            .group("commons-cli")
+            .name("commons-cli")
+            .versionBound(VersionBound.create("1.4", "1.4"))
```

Comment:
Not that it has to be done now, but is there a plan to fetch these hard-coded values from some other file?

Reply:
I thought about that too, but didn't come up with a better way to deal with it :/ This dependency is very specific to Java so it doesn't make sense to configure it from yaml.
Also, commons-cli doesn't have new release very frequently: 1.4 was release two years ago and the previous version before that (1.3.1) was released four years ago. So it wouldn't cause much maintenance friction for us to hardcode the version here.

Reply:
There is the dependencies.yaml file that might be able to be referenced here. We could add another line to that file for the commons-cli version and then fetch from the package config?

Reply:
Oh OK. I'll fix it.